src/test/sitl/sitl_harness.py

```
@@ -0,0 +1,743 @@
+
+    rc.set(6, RC_HIGH)  # AUX3: ANGLE for the manual segment
+    rc.set(4, RC_HIGH)  # AUX1: arm (throttle is low)
+    wait_for("armed", lambda: BOX_ARM in sitl.modes())
+
+    rc.set(2, 1600)     # raise throttle (wasThrottleRaised) and climb clear of the ground
+    time.sleep(3.0)
+
+    rc.set(5, RC_HIGH)  # AUX2: AUTOPILOT
+    modes = wait_for(
+        "AUTOPILOT + ALTHOLD + POSHOLD active (mode wiring)",
+        lambda: sitl.modes() >= {BOX_AUTOPILOT, BOX_ALTHOLD, BOX_POSHOLD} and sitl.modes() or None,
```

Ruff (0.15.20)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)